docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM ubuntu:22.04
```

Comment:
wouldn't it be easier and quicker to use the oficial Varnish instance?

Reply:
I would prefer to have the same installation steps here as we use in CI - this way everything is tested consistently

Reply:
then it would be better to fetch the file from the original repositories rather than duplicate it everywhere

Reply:
not sure what you mean - which file?   `justfile` has the "official" way to fetch and install Varnish - we can keep them in sync quiet easily by simply diffing between multiple vmod repos. Eventually we may introduce a helper cargo tool, e.g. `cargo-varnish` that will add common commands like starting a docker, installing the needed dependencies, etc

Reply:
I mean **this** file

Reply:
you mean Dockerfile should be the same for all vmod files? I agree, but it doesn't have to be identical (e.g. i added `libopenssl-dev`, which is not needed by varnish-rs).  But more importantly, it is not worth reusing a file just yet - easier to copy/paste it between repos for now.  Eventually yes - we should build a tool (see my prev comment)